src/components/chart/_macro-options.md

```diff
@@ -28,6 +28,7 @@
 | footnotes                | `Object<Footnotes>` | false                  | Footnotes to appear below the chart in a 'details' element                                                                                                                                                                                                                                                                                          |
 | fallbackImageUrl         | string              | false                  | A url for a fallback image to display instead of the chart or iframe visualisation if JavaScript is disabled. If a fallback image is passed with an iframeUrl parameter, the iframe will be hidden for non-JavaScript users. If it is not passed, it is assumed the iframe content will provide a fallback image to be displayed inside the iframe. |
 | fallbackImageAlt         | string              | false                  | Alt text for the fallback image - a short description only, as the description field describes the chart for screen readers. Defaults to "Fallback image for the chart as JavaScript is disabled"                                                                                                                                                   |
+| iframeAspectRatio        | string              | false                  | Sets the aspect ratio for iframe charts. Only applicable when using an iframe chart. Acceptable values are 16-9, 4-3, 21-9, 1-1, 3-2, and 9-16. The value must be provided in the x-y format (e.g. 16-9). If an unsupported value is provided, the aspect ratio will default to 16-9. Defaults to 16-9 when not specified.                          |
```

Comment:
I think this could be reworded to something like "Sets the aspect ratio for iframe charts. Only applicable when using an iframe chart. Acceptable values are 16-9, 4-3, 21-9, 1-1, 3-2, and 9-16. The value must be provided in the x-y format (e.g. 16-9). If an unsupported value is provided or a value is not provided, the aspect ratio will default to 16-9."
This avoids the need to say it defaults to 16-9 twice